src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Routes, Link,
+} from 'react-router-dom';
+import BookList from './Components/BookList';
+import Categories from './Components/Categories';
+
+function App() {
+  return (
+    <Router>
+      <div className="App">
+        <header className="header">
+          <h1>Bookstore CMS</h1>
+          <nav className="nav">
+            <ul className="navBar">
+              <li className="nav-link">
+                <Link to="booklist">BOOKS</Link>
+              </li>
+              <li className="nav-link">
+                <Link to="categories">CATEGORIES</Link>
+              </li>
+            </ul>
+          </nav>
+        </header>
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you extract this Navigation bar into a new component (like Navbar/Nav) and render that component here instead to make your code look more neat and modularized.